homeassistant/components/device_tracker/__init__.py

```diff
@@ -208,6 +219,14 @@ def see(self, mac: str=None, dev_id: str=None, host_name: str=None,
             location_name: str=None, gps: GPSType=None, gps_accuracy=None,
             battery: str=None, attributes: dict=None):
         """Notify the device tracker that you see a device."""
+        run_coroutine_threadsafe(
```

Comment:
I don't think that we guarantee that after the see call is done any state has changed. So we should be able to use `fire_coroutine_threadsafe`.